developers/advanced/advanced-pinot-setup.md

```diff
@@ -274,6 +274,30 @@ A TABLE in regular database world is represented as \<TABLE>\_OFFLINE and/or \<T
 
 See [`examples`](https://github.com/apache/pinot/tree/master/pinot-tools/src/main/resources/examples) for all possible batch/streaming tables.
 
+### Add an inverted index to your table automatically
+
+It is possible to force the creation of an inverted index automatically by adding an entry to your [table index config](../../configuration-reference/table.md#table-index-config) in the table configuration file. This works whether you are creating a [batch (offline) table](#batch-table-creation) or a [streaming (real-time) table](#streaming-table-creation) and is accomplished by setting the `createInvertedIndexDuringSegmentGeneration` flag to `true` in your table config, as follows:
```

Comment:
Actually this config is only applicable to batch tables, in the basic setup. In streaming tables, everything is automatically created. Unless you're manually pushing to the streaming table, but that's a special operation anyway